themes/src/main/resources/theme/base/login/passkeys-condional-authenticate.ftl

```diff
@@ -0,0 +1,202 @@
+                                            ${kcSanitize(authenticator.createdAt)?no_esc}
+                                        </span>
+                                    </div>
+                                </div>
+                                <div class="${properties.kcSelectAuthListItemFillClass!}"></div>
+                            </div>
+                        </#list>
+                    </div>
+                </#if>
+            </#if>
+
+            <div id="kc-form">
+                <div id="kc-form-wrapper">
+                    <#if realm.password>
+                        <form id="kc-form-login" onsubmit="login.disabled = true; return true;" action="${url.loginAction}" method="post" style="display:none">
+                            <#if !usernameHidden??>
+                                <div class="${properties.kcFormGroupClass!}">
+                                    <label for="username" class="${properties.kcLabelClass!}">${msg("passkey-autofill-select")}</label>
+                                    <input tabindex="1" id="username"
+                                        aria-invalid="<#if messagesPerField.existsError('username')>true</#if>"
+                                        class="${properties.kcInputClass!}" name="username"
+                                        value="${(login.username!'')}"
+                                        autocomplete="username webauthn"
+                                        type="text" autofocus autocomplete="off"/>
+                                    <#if messagesPerField.existsError('username')>
+                                        <span id="input-error-username" class="${properties.kcInputErrorMessageClass!}" aria-live="polite">
+                                            ${kcSanitize(messagesPerField.get('username'))?no_esc}
+                                        </span>
+                                    </#if>
+                                </div>
+                            </#if>
+                        </form>
+                    </#if>
+                    <div id="kc-form-passkey-button" class="${properties.kcFormButtonsClass!}" style="display:none">
+                        <input id="authenticateWebAuthnButton" type="button" onclick="doAuthenticate([])" autofocus="autofocus"
+                            value="${kcSanitize(msg("passkey-doAuthenticate"))}"
+                            class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonBlockClass!} ${properties.kcButtonLargeClass!}"/>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <script type="text/javascript" src="${url.resourcesCommonPath}/node_modules/jquery/dist/jquery.min.js"></script>
+        <script type="text/javascript" src="${url.resourcesPath}/js/base64url.js"></script>
+        <script type="text/javascript">
```

Comment:
Rather than putting this script in-line, I would prefer it if we load this from a URL. Mostly because it de-clutters the template, and also because it will allow us to run some existing tooling over it in the future.

New scripts should also be loaded using the [`module` type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#applying_the_module_to_your_html), so this would end up looking something like:

```html
<script type="module" src="${url.resourcesPath}/js/passkeys-conditional-authenticate.js"></script>
```